Makefile

```diff
@@ -63,16 +87,14 @@ deps:
 test:
 	go test ./...
 
-# Specify the name for the binaries
-UPGRADE=upgrade
-
 # Specify the name of the docker repo for amd64
 UPGRADE_REPO_NAME_AMD64="upgrade-amd64"
 MIGRATE_REPO_NAME_AMD64="migrate-amd64"
```

Comment:
q: Is this kept for pushing the image from travis?

Reply:
Yes until all the other repos are not multi-arch supported.